shopedia/src/auth/config.php

```diff
@@ -2,7 +2,7 @@
   $user = 'root';
   $password = 'password';
   $db = 'shopedia';
-  $host = 'localhost';
+  $host = getenv('ENV_VAR') == 'docker' ? 'db' : 'localhost';
```

Comment:
Added a flag to check if the webapp is being setup using docker. For Docker, the hostname has to be the same as what was stated in `services` in `docker-compose.yml`